### ISSUE TITLE
KONOBI/Test-FITesque-0.03.tar.gz fails with _071

### ISSUE THREAD

**Issue by andk** [Test-More/test-more#485]
http://www.cpantesters.org/cpan/report/202fce96-63a2-11e4-a548-c8dabb36d621

There are also several unrelated fail reports recently that muddy the waters. Here is only a "Tried to plan twice" issue.

**Comment by andk** [comment]
I might be interested in fixing FITesque. I'm not asking for a complete patch but for a description of what the issues are that need addressing

**Comment by exodist** [comment]
I already submited a patch in the RT queue.

The issue is this: There is a bug in the old Test::Builder that allows you
to plan twice if you call ->no_plan or ->expected_tests. This bug was fixed
in the new Test::Stream stuff. The unit tests for Test::FITesque validate
that a plan is issued, but they do so after a real plan is issued.
Test::Stream fixed the multiple-plan bug, but that broke the Test-FITEsque
tests. My patch works around this issue.
